v2/internal/attacktechniques/k8s/privilege-escalation/hostpath-volume/main.tf

```
@@ -9,7 +9,7 @@ terraform {
 
 locals {
   kubeconfig_path = pathexpand("~/.kube/config")
-  namespace       = format("stratus-red-team-%s", random_string.suffix.result)
+  namespace       = "stratus-red-team-hostpath-namespace-${random_string.suffix.result}"
```

same, let's use format when we can for readability

```suggestion
  namespace       = format("stratus-red-team-hostpath-namespace-%s", random_string.suffix.result)
```